File: benchmark/zaman.c
#include <stdio.h>
int main() {
    long long i = 0;
    long long sinir = 2000000;
    long long toplam = 0;
    while(i < sinir) {
        toplam = toplam + i;
        i = i + 1;
    }
    printf("Bitti\n");
    return 0;
}

File: benchmark/zaman.rs
fn main() {
    let mut i: i64 = 0;
    let sinir: i64 = 2000000;
    let mut toplam: i64 = 0;
    while i < sinir {
        toplam = toplam + i;
        i = i + 1;
    }
    println!("Bitti");
}
